config-example.yaml

```diff
@@ -0,0 +1,11 @@
+meta:
+  version: latest
+  name: my-nats-connector
+  type: nats-source
+  topic: nats-topic
+  create-topic: true
```

Comment:
This field doesn't exist actually, It lives only due to copy-pasting from one example into another :)

Reply:
Yes, I removed it back in the day and I should have known :see_no_evil: